styles/main.css

```diff
@@ -72,8 +74,6 @@ footer hr {
 }
 ul {
     list-style-type: upper-alpha;
-    margin: 0;
-    padding: 0;

```

Comment:
These need to go in the `nav ul` selector above so that ONLY the header list is effected